src/assets/simp-environment/FakeCA/ca.cnf

```diff
@@ -115,7 +115,7 @@ organizationalUnitName_default = Hosts
 
 commonName                     = Common Name (eg, your name or your server\'s hostname)
 commonName_max                 = 64
-commonName_default 		= Fake Org Fake CA - 3c3315f5
+commonName_default 		= Fake Org Fake CA - 72cfa70f
```

Comment:
@lamawithonel No, that changes on every run so that you end up with an appropriate FakeCA name each time.